BitmovinAnalyticsCollector/Classes/util/Util.swift

```diff
@@ -21,7 +23,11 @@ class Util {
         let scale = UIScreen.main.scale
         let height = UIScreen.main.bounds.size.height * scale
         let version = UIDevice.current.systemVersion
+        #if os(iOS)
         let carrier = CTTelephonyNetworkInfo().subscriberCellularProvider?.carrierName ?? "Unknown Carrier"
+        #else
+        let carrier = "Unknown Carrier tvOS"
```

Comment:
Actually this line would also be executed for a OSX version of the collector, but since we just support iOS and tvOS right now we are fine I guess.

Reply:
ill add a check for tvOS as well